docs/en/trusted-solutions/tier1/getting_started/about.md

```diff
@@ -26,9 +27,17 @@ The isolation feature set is a solution that requires less upkeep, as well as le
 * No manual Shotgun updates required
 * Very low level of maintenance required for the AWS components
 
-## What the Isolation Feature Set is not
+## Media isolation feature
+Media Isolation allows your studio to keep the ownership and control of the media and attachments that you upload to Shotgun. With Media Isolation, all the content that you upload to Shotgun can be store in your studio private S3 bucket. Access to the media is provided to the Shotgun service only, using AWS AssumeRole keyless Security Token Service. Your studio remains in control of the assets and the access to the assets, access that you can revoke at will.
```

Comment:
`Your studio remains in control of the assets and the access to the assets, access that you can revoke at will.

May be rephrase it difficult to read, assets assets access access.